docs/Options.md

```diff
@@ -127,6 +127,7 @@ Hostname is shown only when you're connected via SSH unless you change this beha
 ### Directory (`dir`)
 
 Directory is always shown and truncated to the value of `SPACESHIP_DIR_TRUNC`. While you are in repository, it shows only root directory and folders inside it.
+If current directory is write-protected or if current user has not enought rights to write in, a padlock (by default) s displayed as suffix.
```

Comment:
tiny: if current user has not **enough~~t~~** rights to write in **it**, a padlock (by default) **is** displayed as **a** suffix